app/src/main/java/com/example/vre_app/Login.kt

```diff
@@ -70,12 +70,26 @@ class Login : AppCompatActivity() {
 
             val intent = Intent(this, Menu::class.java)
             startActivity(intent)
-        } else {
+        }
+
+        // Using the Create Account Class File (Saving Username and Password)
+        else if(inputUsername == Passing.username && inputPassword == Passing.password){
+            val text = "Successfully Logged In"
+            val duration = Toast.LENGTH_LONG
+            val toast = Toast.makeText(this, text, duration)
+            toast.show()
+
+            val intent = Intent(this, Menu::class.java)
```

Comment:
```suggestion
        if((inputUsername == Passing.username && inputPassword == Passing.password) || 
        (inputUsername == "username" && inputPassword == "password"){
            Toast.makeText(this, "Successfully Logged In", Toast.LENGTH_LONG).show()
            val intent = Intent(this, Menu::class.java)
```
Consider combining this elseif and the previous if block.

Reply:
The else-if statement is much better since it can able to login as a admin or the create account for the user. If you do a if statement it will create an error message. I did that, and it display "Email/Username or Password is incorrect, Try Again" once it login the account.  So I think leave the else if statement so it be much better

Reply:
But arent they doing the same thing? if (username & pw combination correct) -> show toast success and open another activity else -> show toast not success.

Reply:
![image](https://user-images.githubusercontent.com/80478758/196474421-71b1131e-5835-42c0-a591-4d899703409c.png)
This if and elseif basically does the same thing -> make a toast success and start new activity
just combine the if statements into one whole ```if (username= "username" && password = "password" || username=passing.username && password = passing.password) { show toast, open new activity } ```

Reply:
Sorry I forgot to add some thing to the code that why I have an error from the code.. other than that yeah everything works fine from now.